src/csharp/Microsoft.Spark.UnitTest/Sql/ColumnTests.cs

```diff
@@ -363,6 +363,68 @@ public void TestOver()
             }
         }
 
+        [Fact]
+        public void TestIsIn()
+        {   
+            {
+                var expected = new System.Collections.Generic.List<string> {"vararg_1", "vararg_2"};
```

Comment:
Can we use `using` for `System.Collections.Generic.List`?

Reply:
done